lua/nougat/cache/buffer.lua

```diff
@@ -125,6 +125,8 @@ local subscribe = {
     local provider
     if pcall(require, "gitsigns") then
       provider = "gitsigns"
+    elseif vim.fn.exists("*GitGutterGetHunkSummary") > 0 then
+      provider = "vim-gitgutter"
```

Comment:
@linrongbin16 Thanks so much for implementing this.

Would you be willing to add tests for the new codes? If you're not interested in doing it now, I can add them for you.

The tests for `git.status` is located here: https://github.com/MunifTanjim/nougat.nvim/blob/700708b73c037e7800cbb27209247fdd1b6a122f/tests/nougat/nut/git/status_spec.lua

Reply:
hi @MunifTanjim , I add 2 cases in https://github.com/MunifTanjim/nougat.nvim/pull/82/commits/6d31c1e6ea07f45ef7fe74f97fdf890a305b6907, but I'm not sure how should I run the unit test in my local machine?

Reply:
You need to run `./scripts/test.sh nougat/nut/git/status` (only that specific test file) or `./scripts/test.sh` (all test files).